hack/upgrade-test-harness/testdata/upcoming/install.yaml

```diff
@@ -0,0 +1,3045 @@
+---
```

Comment:
Git mishap?

Reply:
Oh yeah I did a `add -A` going to fast. Will fix.